PointOfDispensingApp/src/components/VaccinationEventPage.vue

```diff
@@ -26,21 +26,42 @@ import PatientInfoComponent from './PatientInfoComponent';
 import VaccinationScreeningComponent from './VaccinationScreeningComponent';
 import VaccinationProceedComponent from './VaccinationProceedComponent';
 import VaccinationCanceledComponent from './VaccinationCanceledComponent';
+import config from '../../config.js';
 
   export default {
     name: 'VaccinationEventPage',
     methods: 
     {
+      setVaccinationProceedDecision(decision)
+      {
+        if(decision == true)
```

Comment:
This logic should be updated to avoid the below edge case -- a user could select "Yes" to the screening questions to make the "Proceed w/ Vaccination" selector appear. But then they could select no on one of them after making "Proceed..." appear and still complete the workflow. This def shouldn't be the case and any update to the screening questions should trigger a check/update on the "Proceed..." selector. 

This does, however, bring up an interesting point of what do you do if the user has started to fill in info for those options. I'd defer to you on best followup for this path. 


![image](https://user-images.githubusercontent.com/67079030/101233392-51bc3480-367e-11eb-99fb-17cf3070e091.png)

Reply:
And don't all of these screening questions need to be "Yes" in order for the "proceed" to appear? It currently works that so long as they've been answered (regardless of yes/no) the proceed appears which seems wrong.

Reply:
This would be good to discuss with a few people on the team.  I went back and forth on this in my head quite a bit!
In my mind, it kind of comes down to how strictly we want to try to enforce a certain workflow (and various details associated with it) vs. providing a means for a trained user to document the vaccination process.  

An alternative solution I am thinking about is doing a validation check when the user selects "submit vaccination record" - at that point, we could flag any potential documentation inconsistencies we identify (like "consent form signed?" is marked as "No," but "Proceed with vaccination?" is marked as "Yes."  What do you think about that?  This would be something to capture in a separate card.

I am also thinking that if we end up being overly restrictive, it adds a lot of complexity and could potentially inhibit some edge case that is actually valid (although I can't think of one offhand...I have encountered something like this in other projects).